### DISCUSSION TITLE
Submitted test results but nothing happens

### DISCUSSION THREAD

**Discussion by JennySeidenschwarz** [bdd100k/bdd100k#282] [Q&A]
Hello everyone :)

I just submitted test results to the test server (https://eval.ai/web/challenges/challenge-page/1653/my-submission), but nothing is happening since 1.5 hrs. Can you somehow see if sth is not working correctly?

Best,
![Screenshot 2022-11-03 at 17 40 37](https://user-images.githubusercontent.com/54901570/199781299-e4b24e75-c798-4bad-bc2a-5e78de3da2a3.png)

Jenny

**Comment by JennySeidenschwarz** [comment]
I actually wrote to ramramrakhya81@gmail.com but did not get any answer :(

**Reply by siyuanliii** [reply]
https://eval.ai/web/challenges/challenge-page/1836/overview/ Please try to submit from here

**Reply by pope-bogdan** [reply]
> https://eval.ai/web/challenges/challenge-page/1836/overview/ Please try to submit from here

Can you tell me, please, which test is this challenge is testing?
